src/SiteConfig/ConfigBuilder.php

```diff
@@ -327,6 +327,11 @@ public function mergeConfig(SiteConfig $currentConfig, SiteConfig $newConfig)
         //      find_string: <other-img
         //      replace_string: <img
         // To fix that issue, we combine find & replace as key & value in one array, we merge them and then rebuild find & replace string in the current config
+
+        if (count($currentConfig->find_string) !== count($currentConfig->replace_string)) {
+            return $currentConfig;
+        }
+
```

Comment:
Could you then move the `http_header` definition before that new condition?

Reply:
moved `http_header` definition